big-red-shuttle/AppDelegate.swift

```diff
@@ -13,7 +13,7 @@ import SwiftyJSON
 enum FileReadingError : Error { case fileNotFound }
 
 struct colorPalette{
-    static var red = UIColor(red: 0.98, green: 0.28, blue: 0.26, alpha: 1.0)
+    static var red = UIColor(red:0.83, green:0.29, blue:0.24, alpha:1.0)
 }
```

Comment:
Monica created a UIColor+Shared.swift file to store colors. This should go there but that change can happen after both of you merge your branches.